internal/runners/checkout/checkout.go

```diff
@@ -125,3 +135,36 @@ func (u *Checkout) Run(params *Params) (rerr error) {
 
 	return nil
 }
+
+func setLibcVersion(libcVersion string) error {
+	if libcVersion == "" {
+		return nil
+	}
+
+	if rt.GOOS != "linux" {
+		return locale.NewError("err_libc_version_not_supported", "libc version is only supported on linux")
```

Comment:
Sorry if I missed this the first time, but aren't all the errors in this function supposed to be input errors?

Reply:
Yes, you're right. Thanks for catching that!